astroquery/alma/__init__.py

```diff
@@ -20,6 +20,14 @@ class Conf(_config.ConfigNamespace):
          'https://beta.cadc-ccda.hia-iha.nrc-cnrc.gc.ca'],
         'The ALMA Archive mirror to use.')
 
+    # archive_url = _config.ConfigItem(
```

Comment:
I don't think we need this committed into the codebase after this is merged?

Reply:
you caught me in the middle of a series of commits.  My current solution might be better long term.... ???

Reply:
yeah, realized, but only after I submitted the review